server/controllers/chartsController.js

```diff
@@ -22,12 +22,31 @@ const postProcessData = (data) => {
       processedData[valueLabel] = [newEntry]
     }
   })
+  const largestHorizontalSection = Object.values(processedData).sort(
```

Comment:
I think this is a case where a comment above this line could be helpful. It isn't immediately obvious why we need the largest horizontal section.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/123/commits/bdef2da4a95afc977ece57e85588ca68dd80b215